imagemagick/README.md

```diff
@@ -0,0 +1,16 @@
+# Imagemagick
+
+This is the Imagemagick application, an app able to do many manipulations to images.
+
+For now only available command is `convert` but others should be added later.
+
+## Running the convert command using [run-app-elfloader](https://github.com/unikraft/run-app-elfloader)
+
+To run the convert command, you first need the image you pretend to convert inside in the dynamic-apps/imagemagick directory. After that, change directory into run-app-elfloader and run the following command:
+
+```console
+./run.sh -r path/to/dynamic-apps/imagemagick /usr/bin/convert [image to be converted] [converted image]
+```
+
+After the conversion is finished, you can press `Ctrl+C` to exit.
+
```

Comment:
Add a section on running the convert using Linux (see the [`python README`](https://github.com/unikraft/dynamic-apps/tree/master/lang/python) for reference).